keanu-project/src/main/java/io/improbable/keanu/distributions/discrete/Multinomial.java

```diff
@@ -39,6 +40,7 @@ private Multinomial(IntegerTensor n, DoubleTensor p) {
             p.sum(0).elementwiseEquals(DoubleTensor.ones(n.getShape())).allTrue(),
             "Probabilities must sum to one"
         );
+        TensorValidator.ZERO_CATCHER.validate(p);
```

Comment:
No

Reply:
I have implemented this now - so it's disabled for this specific context. I don't think it would be right to disable `ZERO_CATCHER` globally (even though it's not used anywhere else currently).

    public static final DebugTensorValidator CATEGORY_PROBABILITIES_CANNOT_BE_ZERO = new DebugTensorValidator<>(TensorValidator.ZERO_CATCHER);